distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/ServiceManagerImpl.java

```diff
@@ -596,35 +592,6 @@ public void waitForSourcesToBeAvailable(String restPath, String... sources)
     }
 
     @Override
-    public void waitForAllConfigurations() throws InterruptedException {
```

Comment:
@vinamartin and I are still looking into whether we should re-implement this method or if it can just be removed.